packages/tools/src/stateManagement/segmentation/setActiveSegmentation.ts

```diff
@@ -0,0 +1,15 @@
+import { defaultSegmentationStateManager } from './SegmentationStateManager';
+
+/**
+ * Retrieves the active segmentation representation for a given viewport.
+ *
+ * @param viewportId - The ID of the viewport.
+ * @returns The active segmentation representation, or undefined if not found.
+ */
+export function setActiveSegmentation(
+  viewportId: string,
+  segmentationId: string
```

Comment:
Should this include the segmentation type?  Also, should it be possible to have 1 active of each type of segmentation?  That is, to have both the active labelmap and the active contour.

Reply:
No, in the concept of activeness, only one segmentation (not representation) is active at a time. We can't have the contour of one segmentation active while the labelmap of another segmentation is active.